### ISSUE TITLE
The NVDA update installation dialog is not centered on the screen as it used to be

### ISSUE THREAD

**Issue by k-kolev1985** [nvaccess/nvda#12192]
### Steps to reproduce:
1. Install one of the older NVDA alpha snapshots based on Python 3.8 (I think that is when the problem started occurring).
2. Update it to the last alpha snapshot via its built-in update mechanism.
3. Observe the position on the screen of the dialog that informs us that the update is being installed.

### Actual behavior:
The "Updating NVDA" dialog is not centered on the screen. Instead, it is positioned in the top left area of it.

### Expected behavior:
All NVDA dialogs, including the "Updating NVDA" one, should be centered on the screen.

### System configuration
- Operating system: Windows 10 Pro version 21H1 (build 19043.985), 64-bit, in Bulgarian with all locale settings set to "Bulgarian".
- NVDA version: alpha-23030,a2b469e4, installed, in bulgarian.
- Processor: Intel Core i5-9400F at 2.90GHz.
- RAM Memory: 16.00GB.
- Graphics: MSI GeForce GTX-1050TI Gaming X 4G, 4096MB dedicated memory, desktop resolution set to 1920x1080.
- Sound Card: Realtek ALC892 at Intel Cannon Point PCH.

#### Does the issue still occur after restarting your computer?
Yes, it does.

#### Have you tried any other versions of NVDA? If so, please report their behaviors.
The versions (even alpha snapshots) before the ones based on Python 3.8) do not exhibit this behavior.

#### If add-ons are disabled, is your problem still occurring?
I haven't tested this. But could add-ons cause such behavior?

#### Did you try to run the COM registry fixing tool in NVDA menu / tools?
No. But due to the nature of the issue, I doubt that it will have any positive impact if I do.

**Comment by seanbudd** [comment]
I haven't been able to reproduce the "Updating NVDA"  aspect of this ticket when updating to the latest alpha - can you confirm that this aspect is no longer an issue?

 "Downloading update" should be fixed in #12295

**Comment by k-kolev1985** [comment]
The dialog that shows the update installation progress of NVDA is still not centered on the screen when updating to NVDA version alpha-22370,35ceac29.

The other not centered dialog is the one for downloading the update, but that I will be able to test when the next alpha comes out.

Those two dialogs are the ones not centered (at least I don't know about others).

**Comment by seanbudd** [comment]
Strange, I'm still unable to reproduce this for "Updating NVDA". I have a couple of questions for troubleshooting as I think this might be a display specific wxWidgets issue.

- If you open the NVDA python console and enter `import wx; wx.GetDisplaySize()` what is returned?
- Do you happen to be using multiple monitors? 
- Does the "Check for update" dialog centre properly?

**Comment by k-kolev1985** [comment]
OK, typing:
import wx; wx.GetDisplaySize()
into the NVDA console returns the following:
wx.Size(1920, 1080)
My DPI scaling is at 100%, if that matters.

No, I do not use multiple monitors.

Yes, as far as I could see (it appears only for a short period of time), the "Check for update" dialog is centered.

**Comment by k-kolev1985** [comment]
I see this issue has been closed, but I can still partially reproduce the problem. The dialog for downloading the update is now centered, but the one for updating NVDA (the one that displays during the update installation process) is still not - it appears in the top left side of the screen.

**Comment by seanbudd** [comment]
Would you be able to update the issue description and reopen the issue?

**Comment by k-kolev1985** [comment]
> Would you be able to update the issue description and reopen the issue?

I've updated the title and description. But I don't know how to reopen the issue. I'm not even sure that I have the necessary privilege's to do that.

**Comment by ehollig** [comment]
@seanbudd can this issue be closed now that #14024 is merged?

**Comment by seanbudd** [comment]
I don't think so, from testing #14024, the experience is better but the dialog still does not centres at the start. This couldn't be fixed easily / wasn't worth fixing in my opinion
